commit d0b07e82639d8b5395b7297d94674ff3c006630c (refs/changes/01/3701/4)
Author: John Levon <john.levon@joyent.com>
Date:   2018-03-26T19:31:25+00:00 (1 year, 6 months ago)
    
    OS-6800 bhyve-opts should be bhyve-extra-opts
    Reviewed by: Jerry Jelinek <jerry.jelinek@joyent.com>
    Reviewed by: Mike Gerdts <mike.gerdts@joyent.com>
    Approved by: Mike Gerdts <mike.gerdts@joyent.com>

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index e76c9d65d4..1fd7208820 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -379,13 +379,13 @@ add_lpc(int *argc, char **argv)
 }
 
 int
-add_bhyve_opts(int *argc, char **argv)
+add_bhyve_extra_opts(int *argc, char **argv)
 {
 	char *val;
 	char *tok;
 	char *lasts;
 
-	if ((val = get_zcfg_var("attr", "bhyve_opts", NULL)) == NULL) {
+	if ((val = get_zcfg_var("attr", "bhyve_extra_opts", NULL)) == NULL) {
 		return (0);
 	}
 
@@ -529,7 +529,7 @@ main(int argc, char **argv)
 	    add_ram(&zhargc, (char **)&zhargv) != 0 ||
 	    add_devices(&zhargc, (char **)&zhargv) != 0 ||
 	    add_nets(&zhargc, (char **)&zhargv) != 0 ||
-	    add_bhyve_opts(&zhargc, (char **)&zhargv) != 0 ||
+	    add_bhyve_extra_opts(&zhargc, (char **)&zhargv) != 0 ||
 	    add_vmname(&zhargc, (char **)&zhargv) != 0) {
 		return (1);
 	}
